### DISCUSSION TITLE
Enforce a review feature

### DISCUSSION THREAD

**Discussion by rickysarraf** [photoprism/photoprism#3604] [Q&A]
To my knowledge, currently, PhotoPrism only half heartedly applies the `Review` feature, i.e. only when certain degraded quality conditions meet. Like in here: https://github.com/photoprism/photoprism/issues/1057

What about the other way around. Is there a way to enforce that all `Imports` directly land under `Review` ? And only show up in the main view after `Approval` ? Same should apply for new items that are added to `Originals` and detected on the next `Index` run. There should be a knob to make all such newly detected items to first go via a `Review` cycle.

**Comment by graciousgrey** [comment]
The purpose of the current review feature is to filter out low quality photos or non-photographic content like memes or screenshots. 

Reviewing all new photos is a slightly different use case and would definitely require a setting, as not everyone would want that. But I agree that this could be useful for some workflows. 

For now you could filter by recently added and move the photos you don't want to  the archive.

**Reply by lastzero** [reply]
It might also be possible to abuse the private flag for this or add a similar flag to require a manual review for reasons other than the quality as currently defined.

**Reply by rickysarraf** [reply]
Thank you. You may want to consider this as a Feature Request.
